config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  resources :mechanics, only: [:show] 
+
+  resources :ride_mechanics, only: [:new, :create]
```

Comment:
Don't need the :new, just the :create since the form is on the mechanics show page. Otherwise, these routes look good.